pyproject.toml

```diff
@@ -13,3 +13,10 @@ exclude = '''
   | hack
 )/
 '''
+
+[tool.codespell]
```

Comment:
I don't like that, it feels totally arbitrary to put this unrelated configuration in a python config file.

I recommend to use .codespellrc

Reply:
Up to you.  I am even a little on your side since then it would not require optional `tomli`.

Just 1 more chance to reconsider under the following arguments

- often people prefer avoiding pollution of root directory with per-tool configuration files.
- [you already have the one (and only content) `tool.black` in pyproject.toml](https://github.com/containers/podman/blob/main/pyproject.toml)
  - FWIW codespell is written in Python , if argument "black is written in Python" comes up ;-)

So let me know

Reply:
I don't want this in a random python file, we are not a python project so please remove it from there. I don't think anyone will ever look into this file ever and fine the codespell config there.